apps/nextra/pages/en/build/indexer/indexer-api/indexer-reference.mdx

```diff
@@ -422,33 +407,31 @@ This table maps tokens, standards, and addresses to human readable names.
-| nft_marketplace_v2_current_nft_marketplace_auctions           | TBD              | TBD         | We're deprecating nft_marketplace processor so all the nft_marketplace tables should be deleted |
-| nft_marketplace_v2_current_nft_marketplace_collection_offers  | TBD              | TBD         | We're deprecating nft_marketplace processor so all the nft_marketplace tables should be deleted |
-| nft_marketplace_v2_current_nft_marketplace_listings           | TBD              | TBD         | We're deprecating nft_marketplace processor so all the nft_marketplace tables should be deleted |
-| nft_marketplace_v2_current_nft_marketplace_listings_aggregate | TBD              | TBD         | We're deprecating nft_marketplace processor so all the nft_marketplace tables should be deleted |
-| nft_marketplace_v2_current_nft_marketplace_token_offers       | TBD              | TBD         | We're deprecating nft_marketplace processor so all the nft_marketplace tables should be deleted |
-| nft_marketplace_v2_nft_marketplace_activities                 | TBD              | TBD         | We're deprecating nft_marketplace processor so all the nft_marketplace tables should be deleted |
-| signatures                                                    | 6/21             | TBD         |  |
-| token_activities                                              | 7/22             | TBD         | Replace with token_activities_v2 |
-| token_activities_aggregate                                    | 7/22             | TBD         | Replace with token_activities_v2 |
-| token_datas                                                   | 7/22             | TBD         |  |
-| token_ownerships                                              | 7/22             | TBD         | Replace with token_ownerships_v2 |
-| tokens                                                        | 7/22             | TBD         | Replaced with v2 tables |
+| Table                                                         | Notes |
+| ------------------------------------------------------------- | ----- |
+| address_version_from_move_resources                           | Replace with account_transactions |
+| address_events_summary                                        | To query custom events, you should create a [No-Code Indexer](https://build.aptoslabs.com/docs/no-code-indexing) |
+| address_version_from_events                                   | To query custom events, you should create a [No-Code Indexer](https://build.aptoslabs.com/docs/no-code-indexing) |
+| coin_activities                                               | Replace with fungible_asset_activities |
+| coin_balances                                                 | Replace with current_fungible_asset_balances |
+| coin_infos                                                    | Replace with fungible_asset_metadata |
+| coin_supply                                                   | No replacement|
```

Comment:
up to you if you want to include it but i've replicated the logic here https://github.com/aptos-labs/explorer/blob/main/analytics/apt_supply.sql